libraries/lib-wave-track/WaveTrack.h

```diff
@@ -371,6 +372,74 @@ class WAVE_TRACK_API WaveTrack final
       */
    );
 
+   /*!
+    * @brief Gets as many samples as it can, but no more than `2 *
+    * numSideSamples + 1`, centered around `t`. Reads nothing if
+    * `GetClipAtTime(t) == nullptr`. Useful to acess samples across clip
+    * boundaries, as it spreads the read to adjacent clips, i.e., not separated
+    * by silence from clip at `t`.
```

Comment:
Is this description really what we want?

To be REALLY strictly conservative of program behavior, in the case of not time stretching, it's possible two clips were separated by a tiny gap of one or two sample intervals, but the paint brush might affect both.

And maybe nobody in the history of Audacity actually ever did this.

Reply:
Now that clips can be stretched, we have to choose between constant time or constant sample-count brush radius. The approach I took is the latter. But then, samples of a clip 10 seconds away should not be taken into account. So I went for the simple rule that they have to touch each other, or nothing happens.
I also reckon we won't run into trouble because of this subtle change of behaviour.